quinn-proto/src/connection/streams/mod.rs

```diff
@@ -195,6 +195,7 @@ impl RecvStream<'_> {
 /// Access to streams
 pub struct SendStream<'a> {
     pub(super) id: StreamId,
+    pub(super) index: Option<usize>,
```

Comment:
What happens when the `swap_remove_entry` call introduced in the previous commit invalidates this?